package.json

```diff
@@ -16,7 +16,7 @@
   "license": "GPL-3.0",
   "devDependencies": {
     "@mate-academy/eslint-config": "*",
-    "@mate-academy/scripts": "^0.3.5",
+    "@mate-academy/scripts": "^1.2.8",
```

Comment:
Version of '@mate-academy/scripts' dependency has been updated. Make sure to test and ensure the newer version does not introduce any breaking changes. 😊